docs/source/intervention_models/maternal_malaria_prevention_therapy/index.rst

```diff
@@ -70,24 +68,21 @@ This is considered to be a strong recommendation based on high quality evidence.
 Baseline Coverage Data
 ++++++++++++++++++++++++
 
-.. todo::
-
-  Document known baseline coverage data, using the table below if appropriate
+As IPTp-SP is recommended by the WHO in Malaria endemic ares in Africa for all pregnant women as part of routine ANC, baseline coverage of this intervention exists at variable levels. [Van-Eijk-et-al-2013]_ performed a meta-analysis of coverage of IPTp-SP among various countries in Sub-Saharan Africa up to 2011.
 
 .. list-table:: Baseline coverage data
-  :widths: 15 15 15 15 15
   :header-rows: 1
 
   * - Location
     - Subpopulation
     - Coverage parameter
     - Value
     - Note
-  * - 
-    - 
-    - 
-    - 
-    - 
+  * - Ethiopia
+    - Pregnant women
+    - Proportion of pregnant women covered by IPTp-SP
+    - 0
+    - Not recommended by Ethiopian government due to low malaria burden, as discussed by [Ouedraogo-et-al-2019]_
 
```

Comment:
This is the relevant section of updates for this PR (I had some weird git problem, but the important part here is just the baseline coverage and associated references)

Reply:
Is it because you build this PR from your last branch?